core/tests/puppeteer-acceptance-tests/user-utilities/logged-in-users-utils.js

```diff
@@ -0,0 +1,691 @@
+      throw new Error (
+        'The About Oppia button in About Menu on navbar ' +
+        'does not open the About page!');
+    } else {
+      showMessage(
+        'The About Oppia button in About Menu on navbar ' +
+        'opens the About page.');
+    }
+  }
+
+  /**
+   * Function for click the Browse Our Lessons button in the About page
+   * and check if it opens the Math Classroom page.
+   */
+  async clickBrowseOurLessonsButtonInAbout() {
+    await this.clickOn(browseOurLessonsButton);
+    if (this.page.url() !== mathClassroomUrl) {
+      throw new Error (
+        'The Browse our lesson button does not open the Math Classroom page!');
+    } else {
+      showMessage(
+        'The Browse our lesson button opens the Math Classroom page.');
+    }
+  }
+
+  /**
+   * Function for click the Access Android App button in the About page
+   * and check if it opens the Android page.
+   */
+  async clickAccessAndroidAppButtonInAbout() {
+    await this.clickOn(accessAndroidAppButton);
+    if (this.page.url() !== androidUrl) {
+      throw new Error (
+        'The Access the Android App button does not open the Android page!');
+    } else {
+      showMessage(
+        'The Access the Android App button opens the Android page.');
+    }
+  }
+
+  /**
+   * Function for click the Visit Classroom button in the About page
+   * and check if it opens the Math Classroom page.
+   */
+  async clickVisitClassroomButtonInAbout() {
+    await this.clickOn(visitClassroomButton);
+    if (this.page.url() !== mathClassroomUrl) {
+      throw new Error (
+        'The Visit Classroom button does not open the Math Classroom page!');
+    } else {
+      showMessage(
+        'The Visit Classroom button opens the Math Classroom page.');
+    }
+  }
+
+  /**
+   * Function for click the Browse Our Lessons button in the About page
+   * and check if it opens the Community Library page.
+   */
+  async clickBrowseLibraryButtonInAbout() {
+    await this.clickOn(browseLibraryButton);
+    if (this.page.url() !== communityLibraryUrl) {
+      throw new Error (
+        'The Browse Library button does not open the Community Library page!');
+    } else {
+      showMessage(
+        'The Browse Library button opens the Community Library page.');
+    }
+  }
+
+  /**
+   * Function for click the Browse Our Lessons button in the About page
+   * and check if it opens the Creator Dashboard page in Create Mode.
+   */
+  async clickCreateLessonsButtonInAbout() {
+    await this.clickOn(createLessonsButton);
+    if (this.page.url() !== creatorDashboardCreateModeUrl) {
+      throw new Error (
+        'The Create Lessons button does not open the Creator Dashboard ' +
+        'in Create Mode!');
+    } else {
+      showMessage(
+        'The Create Lessons button opens the Creator Dashboard ' +
+        'in Create Mode.');
+    }
+    await this.page.waitForNavigation();
+    const urlRegex = /http:\/\/localhost:8181\/create\/\w*(\/\w*\/?\w*)?/;
```

Comment:
What does this regex do? I thought it would just be localhost:8181/create/{{something}} -- or is it more complex than that? Maybe add a comment above it with a sample URL.

Reply:
I would recommend making it more specific, actually. What is the actual format of the URL that you expect to end up on (it should be one of those two)? Check for that -- the only part that should be unspecified is the "lhAlObgCmBk6" part, everything else should be hardcoded.